akka-docs/src/main/paradox/typed/persistence-lifecycle.md

```diff
@@ -0,0 +1,61 @@
+# Persistence - lifecycle
+
+# Four behaviors of EventSourcedBehavior
+ 
```

Comment:
Shall I delete all of this since it came from those docs and I'll just delete this new file. All the other content is relocated to the main doc.

Reply:
👍

Reply:
Gone :)